<!-- -->
<launch>
	
	<arg name="file_path" default="$(find recorder)/data/state/" />
	<arg name="file_name" default="imu.txt" />

	<node name="record_IMU_node" pkg="recorder" type="record_IMU_node" output= "screen">
		<param name="file_path" value="$(arg file_path)"/>
		<param name="file_name" value="$(arg file_name)"/>
		<param name="imu_topic" value="/imu" />
	</node>

</launch>
